schema/mysql/v57/cadence/schema.sql

```diff
@@ -226,7 +226,7 @@ CREATE TABLE history_tree (
   tree_id        BINARY(16) NOT NULL,
   branch_id      BINARY(16) NOT NULL,
   --
-  in_progress    BOOLEAN NOT NULL,
+  in_progress    BOOLEAN NOT NULL, -- Deprecated
```

Comment:
i guess this can be deleted

Reply:
could you plz also delete the corresponding `in_progress` in the schema update files?

Reply:
Out of curiosity, wouldn't be safer to delete the column from a new migration?

Reply:
@sevein Are you saying that, if someone use old commit of code but install the schema from this new commit, then it will break. I agree but this usually very rare because people should use the schema files from the same commit I think. But this PR does break some implicit best practice that schema migration changes should be incremental and immutable...

I think we are just do special case here. Because adding one new schema version just for deleting this deprecated column doesn't buy much, only giving us one more version which is not ideal. 

Thanks for your thoughts anyway.

Reply:
Thank you, understood!